src/app/icd/server/ICDConfigurationData.h

```diff
@@ -29,9 +29,16 @@ namespace app {
 // Forward declaration of ICDManager to allow it to be friend with ICDConfigurationData.
 class ICDManager;
 
-// Forward declaration of TestICDManager to allow it to be friend with the ICDConfigurationData.
+// Forward declaration of TestICDManager tests to allow it to be friend with the ICDConfigurationData.
 // Used in unit tests
-class TestICDManager;
+class TestICDManager_TestICDStateObserverOnTransitionToIdleModeGreaterActiveModeDuration_Test;
```

Comment:
Tests need to be able to Modify the Active/Idle mode durations to trigger specfic edges cases.
The API itself is private because we only want the ICDManager to be able to access it to trigger these updates and any associated buisness logic. 

Since this logic isn't there yet, we use the `friend` option for now.